tests/containers/prepare_hpvs.pm

```diff
@@ -100,6 +100,7 @@ sub find_entity {
 
 sub run {
     select_serial_terminal();
+    $ibm->{instance} .= lc(sprintf("%08X", rand(0xFFFFFFFF)));
```

Comment:
Are you creating a new uniq identifier here?

Reply:
yes, we might need to run containers of multiple versions